compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/JIMethodCLICall.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.ballerinalang.compiler.bir.codegen.interop;
+
+import io.ballerina.tools.diagnostics.Location;
+import org.wso2.ballerinalang.compiler.bir.model.BIROperand;
+
+import java.util.List;
+
+/**
+ * Java method invocation call for internal cli argument population modeled as BIR terminator.
+ *
+ * @since 2201.8.0
```

Comment:
```suggestion
 * @since 2201.9.0
```
We can keep this suggestion unresolved and change once the PR is ready to merge.